appveyor.yml

```diff
@@ -12,20 +12,20 @@ for:
       - image: Ubuntu2204
 
   install:
-    - sh: sudo apt-get update && sudo apt-get install -y dotnet-sdk-7.0=7.0.403-1
+    - sh: sudo apt-get update && sudo apt-get install -y dotnet-sdk-8.0
```

Comment:
I would like to use the explicit version `dotnet-sdk-8.0=8.0.100` (or `8.0.100-1` you have to check it). When Microsoft releases a new version our tests will no longer work.

Reply:
From reading the console output, I know it is `8.0.100-1`.

Reply:
The tests will work fine with a later version. Having an explicit version is just extra maintenance

Reply:
If the unit test won't work for newer SDK, it means either our unit test can improve or it is a real problem found.